Inspicio/Controllers/ImagesController.cs

```diff
@@ -17,56 +17,45 @@ namespace Inspicio.Controllers
     [Authorize]
     public class ImagesController : Controller
     {
-        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly UserManager<ApplicationUser> UserManager;   
+        private readonly IHostingEnvironment Environment;
+        private readonly ApplicationDbContext Context;
 
-        IHostingEnvironment _environment;
-
-        private readonly ApplicationDbContext _context;
-
-        public ImagesController(ApplicationDbContext context, IHostingEnvironment _environment, UserManager<ApplicationUser> userManager)
+        public ImagesController(ApplicationDbContext Context, IHostingEnvironment Environment, UserManager<ApplicationUser> UserManager)
         {
-            this._environment = _environment;
-            _context = context;
-            _userManager = userManager;
+            this.Environment = Environment;
+            this.Context = Context;
+            this.UserManager = UserManager;
         }
 
         // GET: Images
         public async Task<IActionResult> Index()
         {
 
             // Only images with the user id set as the owner should be passed into the view
-            var UserID = _userManager.GetUserId(HttpContext.User);
-
-            var allImages =  _context.Images;
-            List<Image> images = new List<Image>();
-
-            foreach (Image image in allImages)
-            {
-                if(image.OwnerId == UserID)
-                {
-                    images.Add(image);
-                }
-            }
+            var UserID = UserManager.GetUserId(HttpContext.User);
```

Comment:
TitleCase for local variables is very very unusual, I don't think I've seen that in a codebase before - your naming style is up to you, but this feels unusual enough to be quite confusing.